src/main.rs

```diff
@@ -33,6 +34,12 @@ struct Args {
     /// Optionally include a commitment level. Defaults to your Solana CLI config file.
     #[clap(global = true, short, long)]
     commitment: Option<String>,
+    /// Optionally include a priority fee, in how many micro lamports you want to pay per compute unit. Defaults to no priority fee. Max is 1 million
+    #[clap(global = true, short, long)]
+    prio_fee: Option<u64>,
+    /// Optionally include the number of compute units you want to pay for. Only works if a priority fee is also set. If prio fee is set, this defaults to 1.4 million
+    #[clap(global = true, short, long)]
+    num_compute_units: Option<u32>,
```

Comment:
Thoughts on updating this interface to take in the amount your'e willing to pay?

Reply:
yeah can back it out from compute units, default it to like 500k. ill change it